style.css

```
@@ -0,0 +1,3 @@
+.header {
+    background-color: black;
```

olha, a cor correta é white

```suggestion
    background-color: white;
```